data/resources/Urban_Jungle/_main.cfg

```diff
@@ -0,0 +1,31 @@
+#textdomain wesnoth-campaign-resources
+[textdomain]
+    name="wesnoth-campaign-resources"
+[/textdomain]
+
+#
+# This is a resource intended for use in select mainline campaigns, and should not be relied upon for UMC.
+# It may change in ways that aren't backwards-compatible with or that negatively affect the balance of UMC content that uses it.
+#
+[resource]
+    id=campaignResource_urban_jungle
+[/resource]
+
+[binary_path]
+    path=data/resources/Urban_Jungle
+[/binary_path]
+{resources/Urban_Jungle/terrain.cfg}
+{resources/Urban_Jungle/terrain-graphics.cfg}
```

Comment:
Okay, next, these lines (the binary path and the include) should be wrapped in an `#ifdef` using the define you declared above.

Reply:
Marking resolved, since you said you aren't able to resolve from VSCode.